bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -182,10 +188,10 @@ data:
         fi
     fi
 
-    if [[ -n "$REDIS_EXTERNAL_MASTER_HOST" ]]; then
-      REDIS_MASTER_HOST="$REDIS_EXTERNAL_MASTER_HOST"
-      REDIS_MASTER_PORT_NUMBER="${REDIS_EXTERNAL_MASTER_PORT}"
-    fi
+    {{- if .Values.replica.externalMaster.enabled }}
+      REDIS_MASTER_HOST={{ .Values.replica.externalMaster.host | quote }}
+      REDIS_MASTER_PORT_NUMBER={{ .Values.replica.externalMaster.port | quote }}
+    {{- end }}
```

Comment:
Please fix the shell indentation after rendering

Reply:
Done